.gitignore

```diff
@@ -19,6 +19,7 @@ docs/jsdoc/
 
 # Logs
 logs/*.log
+logs/**/*.log
```

Comment:
The rule `logs/*` will cover both of above lines. And we don't need anything else in log directory.